package.json

```
@@ -71,5 +75,21 @@
         "protobufjs": "^7.2.4",
         "semver": "^7.5.3",
         "nx": "16.5.5"
-    }
+    },
+    "lint-staged": {
+        "packages/**/src/**/*.ts": [
+          "prettier --write"
+        ],
+        "!(packages/**/src/dist/)*.ts": [
```

No such dir. Did you mean?
```suggestion
        "!(packages/**/dist/)*.ts": [
```

---

Ah, yes you're right. I've typed the wrong dir.